### PR TITLE
fix blink & deny entry

### PR DESCRIPTION
**Pull request by LievenLitaer**
change entry of {tem} as it appears in the qepHom-booklet,
correct autotranslated entry of "blink"

### CODE REVIEW

File: mem-16-t.xml [hunk 1]
```diff
@@ -3850,7 +3850,7 @@ This word takes the suffix {-Du':n:suff} when it refers to a body part, and {-me
       <column name="examples_zh_HK"></column>
       <column name="examples_pt"></column>
       <column name="search_tags">contradict, disavow, claim as false</column>
-      <column name="search_tags_de">widersprechen, ablehnen, als falsch behaupten</column>
```
Comment (De7vID) [inline mem-16-t.xml:3853]: Wouldn't it still be useful to keep the search tags?

**Comment by De7vID** [comment]
This pull request should've been part of #385.

**Comment by De7vID** [comment]
I see now why your previous edit wasn't saved. What you've done here is made a merge request against *your own* branch. You should've edited your branch directly, which would've made your edit a part of PR #385. As it is, PR #385 is merged into master, but *this* change (#388) is just merged to your own branch.

**Comment by De7vID** [comment]
I'm correcting this PR in #389.

File: mem-16-t.xml (hunk 1 above)
Comment (LievenLitaer) [reply mem-16-t.xml:3853]: Sure, why not.

**Pull request merged**